ci-templates/gitlab/README.md

```diff
@@ -35,4 +36,28 @@ Make sure to enable the "Expand variable reference" flag.
 - `T4C_USERNAME` and `T4C_PASSWORD`: Username / password for the T4C repository
 
 This is the minimal configuration. For more advanced configuration options,
-please refer to the [Gitlab CI template](./exporter.yaml).
+please refer to the [Gitlab CI template](./exporter.yml).
+
+## Diagram cache
+
+Please add the following section to your `.gitlab-ci.yml`:
+
+```yml
+variables:
+  CAPELLA_VERSION: 6.0.0 # Enter the Capella version of the model here, only versions >= 6.0.0 are supported
```

Comment:
Is it correct that only 6.0.0 models are supported?

Reply:
Good catch, I've removed it.